doc/basics/compilation.md

```diff
@@ -152,7 +152,7 @@ Out-of source builds are mandatory, we recommend to follow the usual sequence, a
 
 ~~~{.bash}
 cmake . -B build-r -DCMAKE_BUILD_TYPE=Release -C external/install-r/radium-options.cmake -DQt5_DIR=path/to/qt5
```

Comment:
same as above, keeping the user outside of the source tree might be more clearer.